Exoft.Gamification.Api.Common/Models/UpdateUserModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Exoft.Gamification.Api.Common.Models
+{
+    public class UpdateUserModel
+    {
+        public Guid Id { get; set; }
+
+        public string Email { get; set; }
```

Comment:
Some props are missing but lets leave it for now... We would get back to it later